packages/ERTP/src/issuerKit.js

```diff
@@ -14,25 +14,31 @@ import './types-ambient.js';
 
 /**
  * @template {AssetKind} K
+ * @typedef {object} IssuerRecord
+ * @property {string} name
+ * @property {K} assetKind
+ * @property {AdditionalDisplayInfo} displayInfo
+ * @property {Pattern} elementShape
+ */
+
+/**
+ * @template {AssetKind} K
+ * @param {IssuerRecord<K>} issuerRecord
  * @param {Baggage} issuerBaggage
- * @param {ShutdownWithFailure=} optShutdownWithFailure If this issuer fails
- * in the middle of an atomic action (which btw should never happen), it
+ * @param {ShutdownWithFailure | undefined} optShutdownWithFailure If this issuer
+ * fails in the middle of an atomic action (which btw should never happen), it
  * potentially leaves its ledger in a corrupted state. If this function was
  * provided, then the failed atomic action will call it, so that some
  * larger unit of computation, like the enclosing vat, can be shutdown
  * before anything else is corrupted by that corrupted state.
  * See https://github.com/Agoric/agoric-sdk/issues/3434
  * @returns {IssuerKit<K>}
  */
-export const vivifyIssuerKit = (
+const setupIssuerKit = (
+  { name, assetKind, displayInfo, elementShape },
   issuerBaggage,
-  optShutdownWithFailure = undefined,
+  optShutdownWithFailure,
```

Comment:
Please revert this change. I like seeing the JS syntax for optional is the function definition. The type definition may be far away and optionality is often an important part of the function's meaning.